lib/stats/stats-cluster.c

```diff
@@ -208,6 +208,19 @@ stats_cluster_track_counter(StatsCluster *self, gint type)
   return &self->counter_group.counters[type];
 }
 
+StatsCounterItem *
+stats_cluster_get_counter(StatsCluster *self, gint type)
```

Comment:
In the end, this function is not used. Can you drop the patch?

Reply:
I want to keep it as I’ll need it later.